app/cdc_uart_bridge.c

```diff
@@ -0,0 +1,119 @@
+#include <FreeRTOS.h>
+#include <task.h>
+#include <queue.h>
+
+#include "drivers/max14662.h"
+#include "drivers/uart.h"
+#include "drivers/usb/cdc.h"
+#include "stm32l4xx_hal.h"
+#include "cdc_uart_bridge.h"
+
+#define CDC_RX_TASK_STACK_SIZE		512
+#define CDC_RX_TASK_NAME			"CDCrx"
+#define CDC_RX_TASK_PRIORITY		1
+
+#define CDC_TX_TASK_STACK_SIZE		512
+#define CDC_TX_TASK_NAME			"CDCtx"
+#define CDC_TX_TASK_PRIORITY		1
+
+/* Fun test string: 
+</////////////////////////////////////////////////////////////>[##################]
+<2345678901234567890123456789012345678901234567890123456789012>[ABCDEFGHIJKLMNOPQR]
+*/
+
+#define QUEUE_LENGTH    10
+#define ITEM_SIZE       sizeof(struct rx_queue_item)
+
+static struct {
+	bool initialized;
+	StackType_t cdc_rx_task_stack[CDC_RX_TASK_STACK_SIZE];
+	TaskHandle_t cdc_rx_task_handle;
+	StaticTask_t cdc_rx_task_tcb;
+	StackType_t cdc_tx_task_stack[CDC_RX_TASK_STACK_SIZE];
+	TaskHandle_t cdc_tx_task_handle;
+	StaticTask_t cdc_tx_task_tcb;
+	StaticQueue_t cdc_tx_queue;
+	QueueHandle_t cdc_tx_queue_handle;
+	uint8_t cdc_tx_queue_storage[QUEUE_LENGTH * ITEM_SIZE];
+	struct rx_queue_item cdc_rx_queue_item;
+} self;
```

Comment:
Not sure the cdc prefix is needed, also maybe it's possible to move the StackType_t + TaskHandle_t + StaticTask_t into a struct and have 2 structs in the self, then you can do self.rx.task_handle